include/extractor/extractor_callbacks.hpp

```diff
@@ -39,7 +54,7 @@ class ExtractorCallbacks
     // used to deduplicate street names and street destinations: actually maps to name ids
     using MapKey = std::tuple<std::string, std::string, std::string>;
     using MapVal = unsigned;
-    std::unordered_map<MapKey, MapVal, boost::hash<MapKey>> string_map;
+    std::unordered_map<MapKey, MapVal, std::hash<MapKey>> string_map;
```

Comment:
`std::hash` is the default, you can remove this. See http://en.cppreference.com/w/cpp/container/unordered_map